source/quickstart.rst

```diff
@@ -0,0 +1,477 @@
+
+  >>> from pyscf.pbc import gto as pbcgto
+  >>> cell_diamond = pbcgto.M(atom = '''C     0.      0.      0.
+                                        C     .8917    .8917   .8917
+                                        C     1.7834  1.7834  0.
+                                        C     2.6751  2.6751   .8917
+                                        C     1.7834  0.      1.7834
+                                        C     2.6751   .8917  2.6751
+                                        C     0.      1.7834  1.7834
+                                        C     .8917   2.6751  2.6751''',
+                              basis = 'gth-szv',
+                              pseudo = 'gth-pade',
+                              a = np.eye(3) * 3.5668)
+
+Besides the difference in import of the ``gto`` module, the only new attributes are ``Cell.a`` and ``Cell.pseudo``. The former of these (``a``) is a matrix of lattice vectors, where each row denotes a primitive vector, while the latter (``pseudo``) is a crystal pseudo potential (cf. `pyscf/pbc/gto/pseudo/GTH_POTENTIALS <https://github.com/pyscf/pyscf/blob/master/pyscf/pbc/gto/pseudo/GTH_POTENTIALS>`_ for a full list of these), which is allowed to mix with an effective core potential through the ``Cell.ecp`` attribute, cf. `pbc/05-input_pp.py <https://github.com/pyscf/pyscf/blob/master/examples/pbc/05-input_pp.py>`_.
+
+The closeness in the APIs for molecular and crystalline input parsing holds true for mean-field and correlated calculations as well. For instance, an all-electron calculation with k-point sampling at the KS-DFT level using :ref:`density fitting <DF>` (recommended) and a :ref:`second-order SCF algorithm <HF>` requires input on par with a standard KS-DFT calculation for a chemical Hamiltonian (cf. `pbc/21-k_points_all_electron_scf.py <https://github.com/pyscf/pyscf/blob/master/examples/pbc/21-k_points_all_electron_scf.py>`_):
+
+ >>> from pyscf.pbc import dft as pbcdft
+ >>> kpts = cell_diamond.make_kpts([4] * 3) # 4 k-poins for each axis
+ >>> krks_diamond = pbcdft.KRKS(cell_diamond, kpts).density_fit(auxbasis='weigend')
+ >>> krks_diamond.xc = 'bp86'
+ >>> krks_diamond = krks_diamond.newton()
+ >>> krks_diamond.kernel()
+
+Alternatively, mixed density fitting (plane waves & Gaussians) is another option for all-electron calculations, this time at the :math:`\Gamma`-point (cf. `pbc/11-gamma_point_all_electron_scf.py <https://github.com/pyscf/pyscf/blob/master/examples/pbc/11-gamma_point_all_electron_scf.py>`_):
+
+ >>> from pyscf.pbc import scf as pbcscf
+ >>> rhf_diamond = pbcscf.RHF(cell_diamond).mix_density_fit()
+ >>> rhf_diamond.with_df.mesh = [10] * 3  # Tune number of PWs for performance/accuracy balance
+ >>> rhf_diamond.kernel()
+
```

Comment:
I would omit mixed density fitting from a quickstart, it is a more advanced option

Reply:
OK.